docs/manual/java/gettingstarted/WhatsNext.md

```diff
@@ -0,0 +1,22 @@
+# What's Next?
+
+After working through "Getting started with Lagom", follow the path that make the most sense to you:
+
+Learn how to integrate with Eclipse:
+
+* [[Import an existing Maven project into Eclipse|EclipseMavenInt]] 
+* [[Create a new Eclipse project using the Lagom Maven archetype|EclipseMavenNew]]
+* [[Import an existing sbt build into Eclipse|EclipseSbt]]
+    
+Learn how to integrate with Intellij:
+
+* [[Import an existing Maven project into Intellij|intellijMaven]]
+* [[Import an existing sbt build into Intellij|IntellijSbtJava]]
+    
+Learn more about Lagom:
+
+* [[Core concepts|CoreConcepts]]
+* [[Using the service locator|ServiceLocator]]
```

Comment:
I really appreciate this, because I guessed at the appropriate links to add and order. I've removed the link to service locator and added the one for writing Lagom services. The issue about the CoreConcepts file covering some of the same info will have to wait for the larger overhaul of concepts. So, what drove me to the differentiation was just trying to link to existing pages without changing content in the Core Concepts pages until later. And I've noted the order that Ignasi proposes as being logical and will keep this in mind when re-organizing the Core Concepts section.